src/components/TopBarMenu/Categories/Categories.jsx

```diff
@@ -0,0 +1,15 @@
+import "./Categories.scss";
+
+export default function Categories() {
+  return (
+    <>

```

Comment:
You don't need Fragment when you have only one node in the root, in your case you have only one called `ul`.